deepchem/feat/smiles_featurizers.py

```diff
@@ -0,0 +1,254 @@
+"""
+Featurizer implementations used in ChemCeption and Smiles2Vec models.
```

Comment:
Can you add a link to the papers here in the docstring for easy reference?

Reply:
Sure!
I have added them in the docstrings for the corresponding featurizers, but will do so here.